Edits to hhb_.inc.php to make it PHP5 compatible

nixin72 opened the pull request:
This file is the same thing as the hhb_.inc.php, except it works with PHP5. Same code, with the PHP7 specific operators replaced with older ones, and any use of a datatype has been removed. Otherwise identical.

nixin72 committed:
Edits to hhb_.inc.php to make it PHP5 compatible

This file is the same thing as the hhb_.inc.php, except it works with PHP5. Same code, with the PHP7 specific operators replaced with older ones, and any use of a datatype has been removed. Otherwise identical.

divinity76 commented:
thanks! SO quote:
> Do you want the PHP5.5 compatible version of your script though? Since a lot of people are still on PHP<7, it might be good to have it up on GitHub still. 

you certainly got a point, some places can't upgrade to PHP7 yet.

as a precaution, i've checked the code in 84f93c8ef3fcee6893e44459060fef1a73019660 , and found nothing bad/suspicious in it.

there is a stray `$x = 2;` on line 3 though, that should be removed.

also, the array type hint for `function setopt_array($options)`  could actually be restored to `function setopt_array(array $options)` - as you can see here https://3v4l.org/gAod9 , PHP has actually supported array input typehints since PHP 5.1.0

and finally, i think i'd prefer the name `hhb_.inc.php5.php`, but it's not important.

- do you want to it before i merge?

nixin72 committed:
Minor requested changes

Changed the file name to that requested by the owner as well as removed one line of code that was floating around pointlessly.

nixin72 committed:
changed the function header of the setopt_array

Changed it to force an array to be entered for the setopt_array method by adding a type-hint for array to it. Minor changed as requested by owner.

nixin72 commented:
Hey, I've made the changed you've requested. The file has been renamed, the type-hint for array re-added to the setopt_array function and the stray `$x = 3;` has been removed. Thank you so much for this awesome library, it's been amazingly helpful and very easy to use!

divinity76 commented:
looks good. thanks. merged.
